src/koheesio/spark/readers/databricks/autoloader.py

```diff
@@ -87,6 +92,10 @@ class AutoLoader(Reader):
         description="Extra inputs to provide to the autoloader. For a full list of inputs, "
         "see https://docs.databricks.com/ingestion/auto-loader/options.html",
     )
+    schema: Optional[Union[Path, str, StructType]] = Field(
+        default=None,
+        description="Explicit schema to infer the schema of the input files.",
+    )
```

Comment:
unfortunately, schema is a reserved name in Pydantic. You can use 'schema' as alias though. Perhaps do this instead:

```python
schema_: Optional[Union[Path, str, StructType]] = Field(
        default=None,
        description="Explicit schema to infer the schema of the input files.",
        alias="schema",
    )
```

with this, the outward API still uses 'schema' - internally in the class you'll have to use `self._schema` though.